src/plots/xyPlot.ts

```diff
@@ -49,6 +49,8 @@ export class XYPlot<X, Y> extends Plot {
         this._cachedDomainY = _lastSeenDomainY;
         _deltaX = 0;
         _deltaY = 0;
+        _scalingX = 1;
+        _scalingY = 1;
```

Comment:
This is just good practice. It fixed the issue initially, but then the line bellow addressed some edge cases. 

I was not resetting _scalingXY before because I was wrongly assuming that both my callbacks (bellow) are going to be called every time. I was not considering the case where pan and zoom is in just one direction. 

I hope it makes sense